ui/summit-2024/src/pages/Home/components/Hero.tsx

```diff
@@ -62,7 +64,14 @@ const Hero = () => {
               },
             }}
           >
-            Voting Closes {formatISODate(eventCache.eventEndDate)}
+            {eventCache.notStarted
+              ? "Voting Opens " + formatISODate(eventCache.eventStartDate)
+              : eventCache.active
+                ? "Voting Closes " + formatISODate(eventCache.eventEndDate)
+                : !eventCache.proposalsReveal
+                  ? "Results Announced " +
+                    formatISODate(eventCache.proposalsRevealDate)
+                  : "THE RESULTS ARE IN!!!"}
```

Comment:
Might be clearer to have a little function for this:
```
const getVotingText = () => {
  if (eventCache.notStarted) {
    return "Voting Opens " + formatISODate(eventCache.eventStartDate);
  }
  if (eventCache.active) {
    return "Voting Closes " + formatISODate(eventCache.eventEndDate);
  }
  if (eventCache.proposalsReveal) {
    return "THE RESULTS ARE IN!!!"
  }
  return "Results Announced " + formatISODate(eventCache.proposalsRevealDate);
};
```

Sometimes can be hard to follow chained ternary operators, especially when one is negated (`!eventCache.proposalsReveal`)